conversions/decimal_to_any.py

```diff
@@ -65,13 +95,6 @@ def decimal_to_any(num: int, base: int) -> str:
         raise ValueError("base must be >= 2")
     if base > 36:
         raise ValueError("base must be <= 36")
-    # fmt: off
```

Comment:
In general, we should not remove `# fmt: off` and `# fmt: on` directives for psf/black.
It should continue to be the author's prerogative to compress uninteresting details.

Reply:
Yep, I fully intended to move the fmt's, I just must've mistyped

Reply:
Oh, @cclauss, it was dhruvmanila that removed the fmt's in a commit

Reply:
I was not trying to point a finger.  I was just making a suggestion.  The directive (like # noqa and # fmt) are there for a reason.